x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/value_lists/value_list_items.cy.ts

```diff
@@ -0,0 +1,150 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { login } from '../../../../tasks/login';
+import { visit } from '../../../../tasks/navigation';
+import { TOASTER_BODY } from '../../../../screens/alerts_detection_rules';
+import { closeErrorToast } from '../../../../tasks/alerts_detection_rules';
+import {
+  createListsIndex,
+  waitForValueListsModalToBeLoaded,
+  openValueListsModal,
+  importValueList,
+  waitForListsIndex,
+  deleteValueLists,
+  KNOWN_VALUE_LIST_FILES,
+  openValueListItemsModal,
+  searchValueListItemsModal,
+  getValueListItemsTableRow,
+  checkTotalItems,
+  deleteListItem,
+  clearSearchValueListItemsModal,
+  sortByValue,
+  updateListItem,
+  addListItem,
+  selectValueListsItemsFile,
+  uploadValueListsItemsFile,
+  mockFetchListItemsError,
+  mockCreateListItemError,
+  mockUpdateListItemError,
+  mockDeleteListItemError,
+} from '../../../../tasks/lists';
+import {
+  VALUE_LIST_ITEMS_MODAL_INFO,
+  VALUE_LIST_ITEMS_MODAL_TABLE,
+  VALUE_LIST_ITEMS_MODAL_TITLE,
+} from '../../../../screens/lists';
+import { RULES_MANAGEMENT_URL } from '../../../../urls/rules_management';
+
+describe(
+  'Value list items',
+  {
+    tags: ['@ess', '@serverless', '@skipInServerless'],
+    env: {
+      ftrConfig: {
+        kbnServerArgs: [
+          `--xpack.securitySolution.enableExperimental=${JSON.stringify(['valueListItemsModal'])}`,
+        ],
+      },
+    },
+  },
+  () => {
+    beforeEach(() => {
+      login();
+      createListsIndex();
+      visit(RULES_MANAGEMENT_URL);
+      waitForListsIndex();
+      waitForValueListsModalToBeLoaded();
+      importValueList(KNOWN_VALUE_LIST_FILES.TEXT, 'keyword');
+      openValueListsModal();
+    });
+
+    afterEach(() => {
+      deleteValueLists([
+        KNOWN_VALUE_LIST_FILES.TEXT,
+        KNOWN_VALUE_LIST_FILES.IPs,
+        KNOWN_VALUE_LIST_FILES.CIDRs,
+      ]);
+    });
+
+    it('can CRUD value list items', () => {
+      openValueListItemsModal(KNOWN_VALUE_LIST_FILES.TEXT);
+      const totalItems = 12;
+      const perPage = 10;
+
+      // check modal title and info
+      cy.get(VALUE_LIST_ITEMS_MODAL_TITLE).should('have.text', KNOWN_VALUE_LIST_FILES.TEXT);
+      cy.get(VALUE_LIST_ITEMS_MODAL_INFO).contains('Type: keyword');
+      cy.get(VALUE_LIST_ITEMS_MODAL_INFO).contains('Updated by: system_indices_superuse');
+      checkTotalItems(totalItems);
+
+      // search working
+      getValueListItemsTableRow().should('have.length', perPage);
+      searchValueListItemsModal('keyword:not_exists');
+      getValueListItemsTableRow().should('have.length', 1);
+      cy.get(VALUE_LIST_ITEMS_MODAL_TABLE).contains('No items found');
+
+      searchValueListItemsModal('keyword:*or*');
+      getValueListItemsTableRow().should('have.length', 4);
+
+      clearSearchValueListItemsModal('');
+      getValueListItemsTableRow().should('have.length', perPage);
+
+      // sort working
+      sortByValue();
```

Comment:
This is ambiguously named: 
```suggestion
      sortValueListItemsTableByValue();
```